client/src/app/site/motions/components/category-list/category-list.component.html

```diff
@@ -48,7 +48,8 @@ <h2 translate>Categories</h2>
 </mat-card>
 
 <mat-accordion class="os-card">
-    <mat-expansion-panel *ngFor="let category of this.categories" multiple="false">
+    <mat-expansion-panel *ngFor="let category of categories" (opened)="openId = category.id" (closed)="panelClosed(category)"
```

Comment:
It was once for checking if something under a category changed, so you can either have a save button for all changes and one for single changes. Since the one for all changes didn't make it, this is just old code.